build/rhel.Dockerfile

```diff
@@ -28,18 +28,8 @@ RUN go mod download
 COPY . .
 
 # compile workspace controller binaries, then webhook binaries
-RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; elif [[ ${ARCH} == "aarch64" ]]; then export ARCH="arm64"; fi && \
-  CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GO111MODULE=on go build \
-  -a -o _output/bin/devworkspace-controller \
-  -gcflags all=-trimpath=/ \
-  -asmflags all=-trimpath=/ \
-  main.go
-RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; elif [[ ${ARCH} == "aarch64" ]]; then export ARCH="arm64"; fi && \
-  CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GO111MODULE=on go build \
-  -o _output/bin/webhook-server \
-  -gcflags all=-trimpath=/ \
-  -asmflags all=-trimpath=/ \
-  webhook/main.go
+RUN make devworkspace-controller
```

Comment:
It leads to non-critical but still warnings:

```
which: no kubectl in (/opt/rh/go-toolset-1.13/root/usr/bin:/opt/rh/go-toolset-1.13/root/usr/sbin:/opt/rh/go-toolset-1.13/root/usr/bin:/opt/app-root/src/bin:/opt/app-root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
which: no oc in (/opt/rh/go-toolset-1.13/root/usr/bin:/opt/rh/go-toolset-1.13/root/usr/sbin:/opt/rh/go-toolset-1.13/root/usr/bin:/opt/app-root/src/bin:/opt/app-root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
which: no controller-gen in (/opt/rh/go-toolset-1.13/root/usr/bin:/opt/rh/go-toolset-1.13/root/usr/sbin:/opt/rh/go-toolset-1.13/root/usr/bin:/opt/app-root/src/bin:/opt/app-root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
```

Reply:
Okay, should be fixed now!